### ISSUE TITLE
How To Persist Data?

### ISSUE THREAD

**Issue by johnyradio** [mbusb/multibootusb#270]
Hi

My linux distro (puppy) generates a persistence file, to persist my desktop config. 
Can i save the persistence file to the USB?
Where?
The distro needs to be able to find the file during boot.

thx

**Comment by mbusb** [comment]
If I am not wrong, the puppy creates and saves its own persistence file. There is not need to add any persistence from multibootusb.

**Comment by johnyradio** [comment]
"puppy creates and saves its own persistence file. There is not need to add any persistence"

Yes. As I mentioned in my OP, "puppy generates a persistence file". 

The issue is where the file gets saved. I moved on from this iissue.IIRC, I was unable to save Puppy's persistence file to the multiboot partition. 

I resolved this by creating 2 partitions on the flash drive, used puppy's own installer to install puppy to the boot partition, and saved the persistence file to the 2nd partition.
